main.py

```diff
@@ -254,6 +270,26 @@ def random_email(name=None):
            random.choices(EMAIL_DATA, emailChoices)[0][1]
 
 
+def fill_out_education_info(driver):
+    try:
+        driver.find_element_by_xpath(DEGREE_COMPLETION_LABEL + '/option[text()="' + random.choice(GRADUATION_STATUS) + '"]').click()
+        driver.find_element_by_xpath(DEGREE_MAJOR_LABEL + '/option[text()="' + random.choice(DEGREE_MAJORS) + '"]').click()
+        driver.find_element_by_xpath(DEGREE_TYPE_LABEL + '/option[text()="' + random.choice(DEGREE_TYPES) + '"]').click()
+        print('successfully filled out degree information')
```

Comment:
Merged in the latex_resumes. The university is auto-filled from the resume, so luckily we don't have to touch that. I still need to copy the long list of degree majors from the site's dropdown list. 

Because the "major/area of study" field is a dropdown, I need specific strings in order to match the selection (I can't type my own option). To make this easier, once I copy over the list of majors, could you modify the resume generator to only use those major names?

The list of majors will be in "common.py" under the variable name "DEGREE_MAJORS"

Also I can't be sure if its still working after the merge, cause email verification. Hopefully there'll be a fix in a few hours when I have time to work on it.

Reply:
Absolutely, should be an easy fix. It’ll likely auto-populate once that’s done, won’t it?

Reply:
I don't believe it's capable of auto-populating dropdown menus. I'm writing a summary of the auto-fill behavior I've noticed and I'll comment it on your PR

Reply:
what if `make_resume()` returned the university and degree it chose, so that `main.py` can use those same values to “manually” fill those dropdowns? or, alternatively (or simultaneously), `make_resume()` accepted `university` and `job` as optional parameters, so that `main.py` could generate them and pass them along just like it does with `name`, `email`, and `phone`? 

I can experiment with both of those with my own commits, if you prefer.

Reply:
It's probably cleanest to do it all as part of the `fake_identity` dictionary, so all the fake identity info is kept together. Then `main.py()` would send the values to `make_resume` . I think we should keep all of the `fake_identity` entries exactly as they appear in the dropdown menus so we can use that dictionary to fill the form. 

The exact wording from the site is like:

> Bachelors Degree (Bachelor of Science or Arts)
> Aviation/ Aeronautics/ Astronautics/ Aircraft Engineering

which isn't really how you'd word it in a resume, so you could have a separate function just for `make_resume()` that takes the site's wording and matches to something that sounds better, like just:

> Bachelor of Science in Aircraft Engineering

Reply:
We could store degree names in a dict, like:
```
DEGREES = [
  { 'dropdown': 'Bachelors Degree (Bachelor of Science or Arts) Aviation', 'resume': 'Bachelor of Science in Aviation' },
  …
]
```
(or even define a small `degree` object, if we want to be fancy). That way we wouldn’t have to worry about parsing from one to the other.

Reply:
I like the idea of storing everything in `fake_identity` for sure.

Reply:
Ok as I'm looking through `resume_faker.py` , I think we're overcomplicating things. Everything currently in that file is a Bachelor's Degree of some sort, so they all correspond to "Bachelors Degree (Bachelor of Science or Arts)". Simple as that

We could add in a bunch of associates degrees as a separate list if we want to mix up the results, then they would all correspond to  "Associates Degree/Diploma" in the form. I think these degrees are most in line with what Kelloggs would actually be looking for, and they're extremely common degrees, so we could just keep it at that. 

_Thinking about adding "High School Diploma" as well, but I'm realizing that if there isn't a college listed on the resume, the form won't autofill the college. Maybe we can experiment with putting high school names in the resume and seeing if the form still auto-fills the education section_

Reply:
Seems like you’re right about us overcomplicating it. We could add associate’s degrees, but i’m not so sure we should be worrying about a recruiter being able to manually tell the difference between a fake resume and a real one (the bullet points are all gibberish anyway), so i don’t think the specific degree matters much (in my opinion).

Reply:
Yeah let's just keep it to Bachelors Degrees only (which is how it already is) and a random major. So I just need to add in the rest of the majors and this will all be dealt with